.github/workflow/nodejs.yml

```diff
@@ -0,0 +1,27 @@
+name: Node-CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x]
+        os: [ubuntu-latest, macOS-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install, build and test
+        run: |
+          npm ci
+          npm run build --if-present
```

Comment:
Why wouldn't `build` command be first?

Reply:
This is just like [hap-nodejs](https://github.com/KhaosT/HAP-NodeJS/raw/master/.github/workflows/nodejs.yml)